snap/snapcraft.yaml

```diff
@@ -16,29 +16,32 @@ plugs:
 
 environment:
   JAVA_HOME: $SNAP/usr/lib/jvm/java-11-openjdk-amd64
-  PATH: $JAVA_HOME/bin:$PATH
+  PATH: $JAVA_HOME/bin:$PATH:$SNAP
+  KUBECONFIG: $SNAP_REAL_HOME/.kube/config
+  SPARK_HOME: $SNAP
+  SPARK_CONFS: $SNAP/lib/python3.10/site-packages/spark8t/resources/conf/
+  SPARK_USER_DATA: $HOME
+  SPARK8T_VERSION: 0.0.1
```

Comment:
Why do we need this variable ?

Reply:
See later in https://github.com/canonical/spark-client-snap/pull/62/files/8e036a644350a62f175248f181cfd36e24104a7c#r1205100357 :-)

I'm good to actually comment out the variable, with an added comment above (as suggested in the related comment). Or to remove it completely if you Guys don't want this.

We will want to pin down the package version though, I don't think that's avoidable.... And we may not only want to get the latest version of `spark8t` but rahter a 'stable' one (avoiding a chain of broken snap/roc unexpectidly). So I think we'll need some level of "manual" control on this, instead of a fully automated chain....?

Anyway, again, I do whatever you Guys want to have.

Reply:
yes, the point here is that this is a variable that you need at build time (since it refers to the tarball do be downloaded) but not at execution, therefore it is not needed among those env variables.

Reply:
I see your point... However I'm not sure if we can define it within the scope of the corresponding build step (using the `python` plugin)... I'm afraid it has to be declared before we refer to the pkg, I can't just have it defined on the same level, no?

Reply:
I've just checked this doesn't work:
```
parts:
  spark8t:
    python-packages:       
      - https://github.com/canonical/spark-k8s-toolkit-py/releases/download/v0.0.1/spark8t-${SPARK8T_VERSION}-py3-none-any.whl
    build-environment:                                                                                                                                                                                          
      - SPARK8T_VERSION: 0.0.1
```
What else could be the correct location of this variable then, if we don't want to have an ENV_VAR? (I kinda see the reason, it's not really withing the scope of ENV_VARs, I just don't see a conceptually better solution -- may be my ignorance on snaps?)

Reply:
Bah, it's even worse: perhaps it's not the location of the variable but it may not be possible to add variables to the `python` plugin `python-packages` section.

Fine, I'll just remove the whole thing for now and we'll experiment when we put this in 'production'

Reply:
Yes, for the time being we can either hard-code the package name  (which is not too far from the fact that as-is we are hard coding an env-variable for the build anyway) or just drop this and address this later.